docs/en/contributing-to-appium/style-guide.md

```diff
@@ -2,8 +2,16 @@
 
 Thanks for your contribution to Appium! Here are the principles we use when
 writing javascript. Please conform to these so we can merge your pull request
- without going back and forth about style. The main principle is: *make your
- code look like the surrounding code*.
+without going back and forth about style. The main principle is: *make your
+code look like the surrounding code*.
+
+### JavaScript
+
+With the exception of the code that runs on devices ([appium-uiautomator2-server](https://github.com/appium/appium-uiautomator2-server) for 
+Android, [WebDriverAgent](https://github.com/appium/WebDriverAgent) for iOS), Appium is written in [Node.js](). If you are
```

Comment:
url for node.js link?

Reply:
Good catch. I originally did this on an airplane with no interwebs, so had to leave the links blank and go back and fill in. Obviously I missed some. :(